lib/index.js

```diff
@@ -243,7 +243,19 @@ module.exports = function nuts(opts) {
                 res.send(output);
            });
         })
-        .fail(next);
+        .fail(function(error) {
+            // Log the error, but simply return a response as if there is no need to update
+            // anything. Squirrel.Windows is a little brat and will crash if it gets a
+            // 4xx or 5xx response from the update server.
+            console.log(error + ' - version: ' + tag);
+
+            var fakeSha = '1111111111111111111111111111111111111111';
```

Comment:
How about the SHA1 of empty input: `da39a3ee5e6b4b0d3255bfef95601890afd80709` :)